client/src/app/app.component.html

```diff
@@ -1,40 +1,35 @@
 <mat-sidenav-container class="sidenav-container">
   <mat-sidenav #drawer class="sidenav" fixedInViewport [autoFocus]="false">
-    <mat-toolbar>Navigation</mat-toolbar>
-    <mat-nav-list class="sidenav-list" role="navigation">
-
-      <!-- Side navigation links -->
+    </mat-sidenav>
+  <mat-sidenav-content class="whole-toolbar">
```

Comment:
I'm confused by the naming here. There's still lots of `sidenav`, but there's no side navigation bar, right?

Reply:
Maybe we want a `mat-toolbar` instead? https://material.angular.io/components/toolbar/overview

Reply:
fixed